src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -6548,28 +6548,59 @@ internal sealed partial class LazyTranslatedQueryExpression : BaseTranslatedQuer
         protected override IOperation ExpressionImpl => _lazyExpression.Value;
     }
 
-    internal sealed partial class FlowCapture : Operation, IFlowCaptureOperation
+    internal sealed partial class FlowCaptureReference : Operation, IFlowCaptureReferenceOperation
     {
-        public FlowCapture(int id, SyntaxNode syntax, bool isInitialization, ITypeSymbol type, Optional<object> constantValue) :
-            base(OperationKind.FlowCapture, semanticModel: null, syntax, type, constantValue, isImplicit: true)
+        public FlowCaptureReference(int id, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+            base(OperationKind.FlowCaptureReference, semanticModel: null, syntax, type, constantValue, isImplicit: true)
         {
             Id = id;
-            IsInitialization = isInitialization;
         }
 
         public int Id { get; }
-        public bool IsInitialization { get; }
         public override IEnumerable<IOperation> Children
         {
             get
             {
                 return Array.Empty<IOperation>();
             }
         }
+
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitFlowCaptureReference(this);
+        }
+
+        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
+        {
+            return visitor.VisitFlowCaptureReference(this, argument);
+        }
+    }
+
+    internal sealed partial class FlowCapture : Operation, IFlowCaptureOperation
```

Comment:
I'm debating whether this operation should declare its type (`Value.Type` rather than `null`).
Is your thinking that captures are kinda like statements, and so don't have a type?

Reply:
These node do not represent values. 

---
In reply to: [163067644](https://github.com/dotnet/roslyn/pull/24358#discussion_r163067644) [](ancestors = 163067644)